cmake/stm32/common.cmake

```
@@ -61,33 +61,44 @@ function(stm32_print_size_of_target TARGET)
 endfunction()
 
 function(stm32_generate_binary_file TARGET)
+    get_target_property(OUTPUT_NAME ${TARGET} OUTPUT_NAME)
+    get_target_property(BINARY_DIR ${TARGET} BINARY_DIR)
+    set(FILE_NAME "${OUTPUT_NAME}.bin")
+    set(FILE "${BINARY_DIR}/${FILE_NAME}")

```

Could you rename to have something more explicit like this (or if you have a better idea) ?:
```suggestion
    set(GEN_FILE_NAME "${OUTPUT_NAME}.bin")
    set(GEN_FILE "${BINARY_DIR}/${FILE_NAME}")
```

---

Yes, I will make it more explicit